### PR TITLE
M4 styling

### PR DESCRIPTION
**Pull request by blakbox23**
In this part of the project I : 
-Added styling to all the components.

### CODE REVIEW

File: src/books.css [hunk 1]
```diff
@@ -7,8 +10,19 @@
   display: flex;
```
Comment (GzimAsani) [inline src/books.css:10]: - [x] You have used `display: flex` 6 times and this way you are repeating the same rule which falls under the of bad practicing with CSS. Kindly create a class with this function and give to any HTML item you want

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @blakbox23 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 
- Project style looks great :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/books.css (hunk 1 above)
Comment (blakbox23) [reply src/books.css:10]: Hey @GzimAsani, you're right. Thank you.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @blakbox23, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

tenor (1)


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by blakbox23** [comment]
Thank you @BrianSammit

**Pull request merged**